### ISSUE TITLE
Cannot parse selector warning for mixins with 2 or more parameters in selector-max-specificity

### ISSUE THREAD

**Issue by PhilippeProulx** [stylelint/stylelint#2677]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where using the less & operator with nested classes within a mixin that has more than one parameter causes a parsing error.

> Which rule, if any, is this issue related to?

It seems to be related to `selector-max-specificity` since the error will not show up when this rule is not used in the config file.

> What CSS is needed to reproduce this issue?

```less
.foo {
  .myMixin(@a, @b) {
    &.bar {
      color: pink;
    }
  }
}
```
Note: The error does not occur when one parameter or less is used in the mixin.

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "selector-max-specificity": "0,4,0"
  }
}
```

> Which version of stylelint are you using?

`7.11.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint foo.less`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to LESS nested class with & operator in a mixin.

> What did you expect to happen?

no parsing error

> What actually happened (e.g. what warnings or errors you are getting)?

The following error was outputted:

```shell
3:5     Cannot parse selector
```

**Comment by hudochenkov** [comment]
Please, use issue template, and provide your config.

**Comment by PhilippeProulx** [comment]
Thanks for the quick reply, I updated my issue to comply with the template and added more information.

**Comment by jeddy3** [comment]
Can we add a heuristic to [`isStandardSyntaxSelector`](https://github.com/stylelint/stylelint/blob/master/lib/utils/isStandardSyntaxSelector.js) that ignores something like `.myMixin(@a, @b).bar`?

@PhilippeProulx Non-standard syntax support isn't a priority for the core team. Please consider contributing a fix if you'd like to see this issue resolved.

**Comment by jwilsson** [comment]
I'll take a look at this one.

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/issues/2755
